packages/components-react/color-sample-react/src/color-sample.tsx

```diff
@@ -1,23 +1,31 @@
-import type { HTMLAttributes } from 'react';
+import type { CSSProperties, HTMLAttributes } from 'react';
 import { clsx } from 'clsx';
-import { forwardRef } from 'react';
+import { forwardRef, useId } from 'react';
 
-export interface ColorSampleProps extends HTMLAttributes<HTMLDataElement> {
-  value: string;
+interface ColorSampleProps extends HTMLAttributes<SVGSVGElement> {
+  color: CSSProperties['color'];
 }
 
-export const ColorSample = forwardRef<HTMLDataElement, ColorSampleProps>(function ColorSample(props, forwardedRef) {
-  const { children, className, style, value, ...restProps } = props;
+export const ColorSample = forwardRef<SVGSVGElement, ColorSampleProps>(function ColorSampe(props, forwardedRef) {
+  const id = useId();
```

Comment:
`useId()` gebruiken betekent dat je deze component niet goed voor server side rendering kan gebruiken in Next.js

Misschien het handiger om zoiets te doen?
```js
const id = id ? `${id}-label` : crypto.randomUUID();
```

Dan kunnen mensen altijd nog `<ColorSample id={useId() />` doen 🤷‍♂️

Reply:
Is het een idee om op feedback te wachten?

Reply:
yes